src/collective/vdexvocabulary/term.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, value, token=None, title=None, description=None, related=[]):
     def __repr__(self):
         rep = "<%s '%s' at %s>" % (
             self.__class__.__name__,
-            self.token,
+            self.token.decode("utf-8"),
```

Comment:
This should be safe_nativestring ot it might break.

Reply:
I actually "fixed" the wrong spot. `__init__` must make sure that the token is always a nativestring (as specified by ITokenizedTerm). Then no further fiddling is needed in `__repr__`.